### PR TITLE
test(weekend): rewrite weekend end to end tests

### PR DESCRIPTION
**Pull request by jniles**
The weekend tests have posed us issues in the past, but they have been updated for simplicity and clarity.  The grid uses GridRow to find entries by their labels and renames variables that were spelled in
correctly throughout the weekend configuration.  Additionally, the client-side has been updated to the latest ES2015+ standards.

### CODE REVIEW

File: client/src/modules/payroll/modals/payroll.modal.html [hunk 1]
```diff
@@ -13,80 +13,66 @@
...
-      </bh-date-interval>
-    </div>
+    <bh-date-interval
+      label="FORM.LABELS.PERIODES"
+      date-id="periodes-date"
+      date-from="PayrollConfigModalCtrl.payroll.dateFrom"
+      date-to="PayrollConfigModalCtrl.payroll.dateTo"
+      required="true"
+      validation-trigger="PayrollModalForm.$submitted">
+    </bh-date-interval>
```
Comment (mbayopanda) [inline client/src/modules/payroll/modals/payroll.modal.html:31]: 👍 good catch and more clean

File: test/end-to-end/payrollConfig/payroll_config.spec.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const chai = require('chai');
 /** configuring helpers* */
 helpers.configure(chai);
 
-describe('Payroll Configuration Management', () => {
+describe.only('Payroll Configuration Management', () => {
```
Comment (mbayopanda) [inline test/end-to-end/payrollConfig/payroll_config.spec.js]: I don't think we need the .only, could you remove it ?

**Review by mbayopanda — COMMENTED** [review]
Tests passes 👍 

Could you remove the `.only` in describe of the test and after that we will bors it

File: test/end-to-end/payrollConfig/payroll_config.spec.js (hunk 2 above)
Comment (jniles) [reply test/end-to-end/payrollConfig/payroll_config.spec.js]: Whoops!  You are right.

**Pull request merged**